### PR TITLE
Fix Resvg image format

### PR DESCRIPTION
**Pull request by ferdnyc**
resvg should be rendering to `QImage::Format_ARGB32_Premultiplied`, according to @RazrFalcon.

Also, tightened up the code a bit. Works fine on my system, but the AppImage will be the ultimate test.

### CODE REVIEW

**Comment by jonoomph** [comment]
LGTM from what I can tell :+1:

**Comment by ferdnyc** [comment]
Retrying build, now that ppa.launchpad.net is hopefully up from its nap.

**Comment by ferdnyc** [comment]
This has been confirmed effective in fixing our AppImage color inversion bug, so merging to develop... (it's already in the release PR)

File: cmake/Modules/FindRESVG.cmake [hunk 1]
```diff
@@ -94,9 +94,23 @@ find_package_handle_standard_args(RESVG
...
+    # Linux needs SHARED to link because libresvg has no SONAME
+    add_library(RESVG::resvg SHARED IMPORTED)
+    set_property(TARGET RESVG::resvg APPEND PROPERTY
+      IMPORTED_NO_SONAME TRUE)
+    message(STATUS "  SHARED IMPORTED target with IMPORTED_NO_SONAME")
+  endif()
+
+  message(STATUS "  INCLUDE_DIRECTORIES: ${RESVG_INCLUDE_DIRS}")
+  message(STATUS "  COMPILE_DEFINITIONS: ${RESVG_DEFINITIONS}")
+  message(STATUS "  IMPORTED_LOCATION: ${RESVG_LIBRARIES}")
```
Comment (ferdnyc) [inline cmake/Modules/FindRESVG.cmake]: GAAAH. I meant to take those out before merging.

**Pull request merged**